src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/ErrorNotification.java

```diff
@@ -73,4 +77,12 @@ public String getErrorDescription() {
     public Timestamp getCreatedAt() {
         return createdAt;
     }
+
+    public void setDocumentControlNumber(String documentControlNumber) {
+        this.documentControlNumber = documentControlNumber;
+    }
+
+    public void setReferenceId(String referenceId) {
```

Comment:
let's add those new params to a constructor so that we can create the object in single step.